docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,112 @@
+    ```
+
+5. Provide your Function logic in the `handler.js` file. In the following example you do not find an actual sanitisation logic, `sanitise` Function is just a placeholder:
+
+   ```js
+   const { v4: uuidv4 } = require('uuid');
+   module.exports - {
+       main: function (event, context) {
+           let sanitsedData = sanitise(event.data)
+           var eventOut=event.buildResponseCloudEvent(uuidv4(), "sap.kyma.custom.acme.payload.sanitised.v1", sanitisedData);
+           eventOut.source="kyma"
+           eventOut.specversion="1.0"
+           event.publishCloudEvent(eventOut);
+           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
+           return eventOut;
+       }
+   }
+   let sanitise = (data)=>{
+       console.log(`sanitising data...`)
+       console.log(data)
+       return data
+   }
+   ```
+   Please note the `sap.kyma.custom.acme.payload.sanitised.v1`. This is the event type the emitter publishes to. The one used here is an example. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](https://kyma-project.io/docs/kyma/latest/05-technical-reference/evnt-01-event-names) page.
+
+   Please note that event object provides convinience functions to build and publish events. To send the event you need to build the Cloud Event. To learn more please visit the [Function's Specification](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-08-function-specification#event-object-sdk) page. In addition, your `eventOut.source` key needs to point at `“kyma”` to use Kyma In-cluster Eventing.
+
+6. Apply your emitter Function:
+
+    ```bash
+    kyma apply function
+    ```
+   Having applied it, your Function is built and deployed in Kyma runtime. Kyma will expose it via API Rule. Any incoming payload would be processed by your emitter Function. It sends the sanitised content to whatever workload that subscribes to selected event type - in our case - receiver Function.
+
+7. Go to your `receiver` folder and run the `init` Kyma CLI command to initiate your function:
+   ```bash
+   kyma init function
+   ```
+8. The `init` command creates the same files as in the emitter folder.
+9.  In the `config.yaml` file configure your Subscriptions to allow for receipt and storage of the event data:
+    ```yaml
+    name: event-receiver
+    namespace: default
+    runtime: nodejs14
+    source:
+       sourceType: inline
+    subscriptions:
+       - name: event-receiver
+         protocol: ""
+         filer:
+            filters:
+               - eventSource:
+                   property: source
+                   type: exact
+                   value: ""
+                eventType:
+                   property: type
+                   type: exact
+                   value: sap.kyma.custom.acme.payload.sanitised.v1
+    ```
+10.  When you call your emitter Function, your receiver Function must also respond.
```

Comment:
I would add a step (10) to deploy the second function ( kyma deploy function ).
After you call it function is configured, built and deployed in kyma runtime. The susbscription becomes active and all events with selected tyle will be processed by the function

11. Test the whole setup

Now when you send a payload to the first function ( i.e via above mentioned POST request ) you should be able to observe that thanks to the fact that functions are joined by the in-cluster eventing  the payload visits both of your functions. 
You can see ( i.e in the function logs ) that both sanitization logic ( by the first function)  and the storing logic ( by the second function ) is executed.

Summary:

You can distribute application logic into specialised, event-driven functions